modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -106,11 +110,30 @@ export const formParts = ({
       const { isValid } = useFormState()
       const { setValue } = useForm()
 
+      const contractAragonAcl = ContractAragonAcl.useRpc()
+      const sdvtRegistry = ContractSDVTRegistry.useRpc()
+
+      const checkIsAlreadyManager = async (address: string) => {
+        const MANAGE_SIGNING_KEYS_ROLE =
+          await sdvtRegistry.MANAGE_SIGNING_KEYS()
+        const result = await contractAragonAcl.getPermissionParamsLength(
+          address,
+          sdvtRegistry.address,
+          MANAGE_SIGNING_KEYS_ROLE,
+        )
+        return !result.isZero()
+      }
+
+      const checkIsLidoRewardAddress = async (address: string) => {
+        const lidoRewardAddress = await sdvtRegistry.getLocator()
+        return address === lidoRewardAddress
+      }
+
       useEffect(() => {
         update(0, { count: `${NOCounts?.current}` })
       }, [setValue, fieldNames.nodeOperators, update, NOCounts])
 
-      const handleAddNodeOperator = useCallback(() => {
+      const handleAddNodeOperators = useCallback(() => {
         const nodeOperators = getValues().SDVTNodeOperatorsAdd
```

Comment:
Please use `fieldsArr` instead of `getValues` here

Reply:
do you kwon the way how to do it?
![image](https://github.com/lidofinance/easy-track-ui/assets/3354239/89f644fc-c83f-4a7a-9808-8ecb607b92cf)
![image](https://github.com/lidofinance/easy-track-ui/assets/3354239/286baa86-d2f7-4247-aaee-72105c0354cd)
Also it is not input operation, so it will not triggered often

Reply:
You're right! Nvm then